src/components/Capture/capture.js

```diff
@@ -191,14 +164,14 @@ class Capture extends Component {
   }
 
   render ({method, side, validCaptures, useWebcam, unprocessedCaptures, ...other}) {
-    const useCapture = (!this.state.uploadFallback && useWebcam && this.supportsWebcam() && isDesktop)
+    const useCapture = (!this.state.uploadFallback && useWebcam && isDesktop)
```

Comment:
should we check here if webcam  is supported, from the exported boolean of onfido react webcam, in order to minimise flickering?